profiler/lib.profiler/build.xml

```diff
@@ -33,19 +33,19 @@
     <target name="compile-system" depends="init">
         <mkdir dir="${build15.classes.dir}"/>
         <javac srcdir="${src15.dir}" destdir="${build15.classes.dir}" debug="${build.compiler.debug}" debuglevel="${build.compiler.debuglevel}" encoding="UTF-8"
-                deprecation="${build.compiler.deprecation}" optimize="${build.compiler.optimize}" source="1.5" target="1.5" includeantruntime="false">
+                deprecation="${build.compiler.deprecation}" optimize="${build.compiler.optimize}" source="1.6" target="1.6" includeantruntime="false">
```

Comment:
I think these are (by far) the most tricky changes - these parts of profiler run on the "target" JDK, AFAIK, so this effectively means no support for JDK 5 and older. Unlike the case with probes, the code that is being compiled here is more complex, and so the amount of tricks we could play is much smaller.

Reply:
had used ${javac.source} instead in recent commit